src/main.c

```diff
@@ -28,8 +29,16 @@
 #include <zephyr/usb/usbd.h>
 
 #include "ulog.h"
+#include "ulog_altitude.h"
+#include "ulog_baro.h"
 #include "ulog_gyro.h"
 
+#define ALTITUDE_SOURCE_TYPE_BARO 1
+
+// Value derrived from BMP280 datasheet (page 15 of 49) for the given sensor
+// configuration.
+#define BMP280_ALTITUDE_VARIANCE_M .017f
```

Comment:
Generally, we wouldn't want sensor or chip-specific things at this high of a level, but it will suffice for now.

Reply:
Perhaps we should think about creating maps between sensor ids and their properties (such as variance) soon.